ml-algorithms/src/main/java/org/opensearch/ml/engine/encryptor/EncryptorImpl.java

```diff
@@ -34,6 +34,7 @@
 @Log4j2
 public class EncryptorImpl implements Encryptor {
 
+    public static final String MASTER_KEY_NOT_READY_ERROR = "ML encryption master key not initialized yet. Please retry after 10 seconds.";
```

Comment:
How about: `The ML encryption master key has not been initialized yet. Please retry after waiting for 10 seconds.`